### ISSUE TITLE
Setup a build process

### ISSUE THREAD

**Issue by keithbloom** [TardisBank/TardisBank#21]
Could be

- https://www.appveyor.com/
- https://azure.microsoft.com/en-gb/services/devops/

Both have a free tier for OS

**Comment by mikehadlow** [comment]
GitHub actions looks interesting too: 
https://github.com/features/actions

**Comment by mikehadlow** [comment]
I've applied. Been added to the queue, so let's see what happens.

**Comment by keithbloom** [comment]
Will it give us a CI server or do we need that anyway?

**Comment by mikehadlow** [comment]
Yes, that's the way I understand it. It'll allow you to run CI tasks on the GitHub infrastructure, then push to a cloud platform somewhere. It looks very cool. I'm looking forward to having a play with it :)
